local_units/management/commands/import-local-units-csv.py

```diff
@@ -0,0 +1,46 @@
+from django.db import transaction
+import csv
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.gis.geos import Point
+
+from api.models import Country
+from ...models import LocalUnit, LocalUnitType
+
+
+class Command(BaseCommand):
+    help = "Import LocalUnits data from CSV"
+    missing_args_message = "Filename is missing. Filename / path to CSV file required. Required headers in CSV: distr_code, GOadm2code, ADM2"
+
+    def add_arguments(self, parser):
+        parser.add_argument('filename', nargs='+', type=str)
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        filename = options['filename'][0]
+        with open(filename) as csvfile:
+            reader = csv.DictReader(csvfile)
+            for row in reader:
+                unit = LocalUnit()
+                unit.country = Country.objects.get(iso3=row['ISO3'])
+                unit.type = LocalUnitType.objects.get_or_create(
```

Comment:
@willemarcel can we do a `print` here if a new type was created? I think it will be useful to warn / inform users when a new type is created so that if it was created accidentally / due to some sort of spelling mistake / typo, they are able to manually correct.

Reply:
@batpad I've added it